extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetTableLocation.java

```diff
@@ -197,6 +197,10 @@ private RowSet computeIndex() {
 
         for (int rgi = 0; rgi < rowGroups.length; ++rgi) {
             final long subRegionSize = rowGroups[rgi].getNum_rows();
+            if (subRegionSize == 0) {
```

Comment:
Does this address empty row groups in the middle of a file? It sort of looks like it does.